drivers/counter/counter_mcux_lpc_rtc.c

```diff
@@ -255,9 +255,11 @@ static DEVICE_API(counter, mcux_rtc_driver_api) = {
 			DT_INST_IRQ(id, priority),				\
 			mcux_lpc_rtc_isr, DEVICE_DT_INST_GET(id), 0);		\
 		irq_enable(DT_INST_IRQN(id));					\
-		if (DT_INST_PROP(id, wakeup_source)) {				\
-			EnableDeepSleepIRQ(DT_INST_IRQN(id));			\
-		}								\
+		IF_ENABLED(CONFIG_PM, (						\
+			if (DT_INST_PROP(id, wakeup_source)) {		        \
+				EnableDeepSleepIRQ(DT_INST_IRQN(id));           \
+			}							\
+		))								\
```

Comment:
Still mixing spaces and tabs (same for the others) try to keep it consistent with existing lines and only use tabs

<img width="187" height="90" alt="image" src="https://github.com/user-attachments/assets/d207451a-0b6c-4b0f-9016-e524e873a6cd" />